calcOrbWeights.jl

```diff
@@ -118,6 +118,25 @@ if emittedParticleHasCharge
     diagnostic_filepath = ""
 end
 
+emittedParticleIsExcited = false #!#
+RHEEP = ["4He-9Be", "4He-10B"] #!# 'reaction has excited emitted particle'
+if reaction in RHEPWC #!#
+    emittedParticleIsExcited = true #!#
+    if product_state === nothing

```

Comment:
yes, but perhaps the user might leave "nothing" as the input variable. In that case we want product_state='GS' such that an error is thrown later and the user knows what to fix from thereon.

Reply:
Alright, good point. But let's not mention anything about 'nothing' in the start file. Since that should only be able to be set as input by mistake etc